packages/manager/modules/log-to-customer/src/live-tail/controller.js

```diff
@@ -0,0 +1,39 @@
+import { LOG_KEYS, DATA_PLATFORM_GUIDE, LOGS_INFO } from '../constants';
+
+export default class LogToCustomerCtrl {
+  /* @ngInject */
+  constructor($translate, LogToCustomerService, coreConfig, coreURLBuilder) {
+    this.$translate = $translate;
+    this.LOG_KEYS = LOG_KEYS;
+    this.LogToCustomer = LogToCustomerService;
+    this.user = coreConfig.getUser();
+    this.coreURLBuilder = coreURLBuilder;
+  }
+
+  $onInit() {
+    this.loading = true;
+
+    this.links = {
+      logPlatformGuide:
+        DATA_PLATFORM_GUIDE[this.user] || DATA_PLATFORM_GUIDE.DEFAULT,
+      logInfo:
+        LOGS_INFO[this.service]?.[this.user.ovhSubsidiary] || LOGS_INFO.DEFAULT,
+      createAccount: this.coreURLBuilder.buildURL(
+        'dedicated',
+        '#/dbaas/logs/order',
+      ),
+      createStream: this.coreURLBuilder.buildURL('dedicated', '#/dbaas/logs'),
+    };
+
+    this.hasAtLeastOneDataStream = false;
+
+    this.LogToCustomer.getDataStreams()
+      .then(({ streams, streamCount }) => {
+        this.streams = streams;
+        this.hasAtLeastOneDataStream = streamCount > 0;
```

Comment:
Why do you need that property ? 
`streamCount` can be different than `streams.length` ?

Reply:
streams is an array of stream by ldp account : 
[
 "ldpAcc1" : [], // Can be empty
 "ldpAcc2": [ {stream1}, {stream2}]
]

This is why I count streams in the arrays while constructing this array on the service. You can have ldp account without any stream in it